includes/render.php

```diff
@@ -61,7 +61,7 @@ class GEM_Form_Fields {
 
 	private static $cycle = 0;
 
-	public function dispatch_field( $field, $cycle = 1 ) {
+	public static function dispatch_field( $field, $cycle = 1 ) {
```

Comment:
Do these need to be static? Would it be better to change the way they're called, instead of making them static?